src/renderer/Components/ReceivingPackage/index.jsx

```diff
@@ -129,31 +99,46 @@ function ReceivingPackage() {
             .map((_, index) => (
               <NumberInput
                 key={index}
-                onKeyDownFunc={(event) => {
-                  validateNumber(event);
-                  updateInputNumbers(event);
-                }}
-                onChangeFunc={handleFocusShift}
+                setInputNumbers={setInputNumbers}
+                inputNumbers={inputNumbers}
+                index={index}
               />
             ))}
         </div>
-        <button
-          type="submit"
-          className="m-5 w-1/3 transform rounded-full bg-slate-200 p-5 text-3xl shadow-lg transition duration-300 hover:scale-105"
-        >
+        <button type="submit" className="button-slate-round">
           받기
         </button>
-        <Modal isOpen={isModalOpen} onClose={navigateToMainPage}>
-          <h2 className="mb-4 text-xl font-semibold">DELIORDER</h2>
-          <p>{modalMessage}</p>
+      </form>
+      <Modal title={"오류"} isOpen={isInfoModalOpen} onClose={closeInfoModal}>
+        <p>{infoMessage}</p>
+      </Modal>
+      <Modal
+        title={"패키지 내용 확인"}
```

Comment:
해당 title 속성값이 문자열이라서 중괄호가 필요 없을 것 같습니다!.😄
```suggestion
        title="패키지 내용 확인"
```

Reply:
또한 일관적이지도 않게 처리했었네요..! 수정되었습니다.!